build-system/release-tagger/test/make-release.test.js

```diff
@@ -21,6 +21,10 @@ test('create', async (t) => {
     '<code>3abc</code></a> - Update packages';
 
   const packages = getExtensionsAndComponents().map((e) => e.extension);
+  const packageChanged = new Set(['bento-accordion']);
+  const packagesNotChanged = packages.filter(
+    (package) => !packageChanged.has(package)
+  );
```

Comment:
But in this test, this package IS changed and should be listed in one of the `<details></details>` portion

Reply:
Here's the body being generated. This seems correct to me? The component IS changed and thus is listed in the "npm packages" section where there were none before. Because the package WAS changed it is also NOT listed in the "Packages not changed" section.

Seems like arbitrarily making a change to `pr2` wasn't a good strategy because of the `renovate-bot` check. Moving the change to `pr1` seems to have resulted in the details section being generated.